bridge/pier/checkpoint.go

```diff
@@ -280,6 +280,12 @@ func (c *Checkpointer) sendRequest(newHeader *types.Header) {
 
 	start := expectedCheckpointState.newStart
 	end := expectedCheckpointState.newEnd
+
+	if bufferedCheckpoint != nil && bufferedCheckpoint.proposer.Equals(hmtypes.BytesToHeimdallAddress(helper.GetAddress())) && bufferedCheckpoint.start == start {
+		c.Logger.Info("Checkpoint already proposed by current validator with same start block", "validatorAddress", bufferedCheckpoint.proposer, "startBlock", start)
```

Comment:
Might be a good idea to trim this log and more checkpoint info
```suggestion
		c.Logger.Info("Checkpoint already proposed to heimdall by us", "proposer", bufferedCheckpoint.proposer, "startBlock", start,"endBlock",end)
```